src/main/java/net/torocraft/signedit/SignEdit.java

```diff
@@ -38,7 +55,7 @@ public void editSign(RightClickBlock event) {
       return;
     }
 
-    if (!isHoldingSign(event.getEntityPlayer())) {
+    if (isHoldingEditor(event.getEntityPlayer())) {
```

Comment:
was the `!` removed by accident?

Reply:
I have literally no idea how that happened....